docs/guide/basics/release-cycle.md

```diff
@@ -12,9 +12,35 @@ So assuming next version is 1.x the two-month cycle will look as following:
 - v1.x-RC.y - unstable version with cutting-edge features ready to test and feedback
 - v1.x.y - stable version of the software **ready for production use**.
 
-# How new features are merged
+## How new features are merged
 
 During RC features Pull Request with new features after feedback and acceptance are normally merged to `develop` branch. 
 After entering the Stabilization Phase we are tagging current develop branch, creating a `RC-x` (where `x` is a number of current version) branch from it and working on stabilization there. 
 During the stabilization phase new features are merged to develop branch and will be merged on next `RC` phase.
 
+## Release cycle flow
+
+### 1. Development phase
+
+The first phase of cycle we're mostly focusing on features and improvements. Branches in that phase should be created from actual `develop`, also PRs should be pointed to this branch. Changes merged to develop are available to test on https://test.storefrontcloud.io
+
+![Development phase](./assets/release-cycle-1.png)
+
+### 2. Release Candidate phase
+
+At some point, when milestone for next minor versions is completed, we're creating new branch from `develop` called `RC-x.y` (example: `rc-1.9`).
+After that new branch is tagged as first RC for version (example `v1.9.0-RC.1`). Then it's ready for testing by community.
+During tests, feedback and stabilisation there could be multiple RC releases on this branch. When improvement is made on this phase, then branch should be created from actual `rc-x.y` and should not contain features at this point - only improvements for current release.
+When release branch is updated, then should `develop` should be synchronized with it.
+
+![Release Candidate phase](./assets/release-cycle-2.png)
+
+### 3. Release phase
+
+When RC version is stable, then release branch is merged into `master` and tagged as next stable version (example `v1.9.0`). After that currently merged release branch is deleted and starts new development phase. Yet, it some critical bug is discovered on current stable version, then new branch should be created from actual `master` and merged into `hotfix-x.y.z` (where `z` is next path intertion; example: `hotfix-1.9.1`). After merging and testing hotfixes, this branch (like release branch before) is merged into `master` and tag by next path version from hotfix branch name.
```

Comment:
Same as releases, I think we should stick with git-flow naming and just refer to semantic versioning description, instead of trying to describe here what the patch version is.
```suggestion
When RC version is stable, then release branch is merged into `master` and tagged as next stable version (example `v1.9.0`). After that currently merged release branch is deleted and starts new development phase. If some critical bug is discovered on current stable version, then new hotfix (for example `hotfix/v1.9.1`, following [Semantic Versioning rules](https://semver.org/)) branch should be created from actual `master`. After merging and testing hotfixes, this branch (like release branch before) is merged into `master` and tagged following [Semantic Versioning rules](https://semver.org/).
```